terraform/application/config/qa.tfvars.json

```diff
@@ -1,6 +1,6 @@
 {
     "cluster": "test",
     "namespace": "bat-qa",
-    "deploy_azure_backing_services": false,
-    "enable_postgres_ssl": false
+    "enable_postgres_ssl": true, 
+    "inf_vault_name": "s189t01-faltrn-dv-inf-kv"
```

Comment:
This is not a terraform variable, it shouldn't be added here. If it's only for github actions, we can create a SLACK_WEBHOOK github repository secret instead